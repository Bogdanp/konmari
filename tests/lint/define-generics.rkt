#lang racket/base

(require racket/generic)

(provide
 gen:to-jsexpr
 ->jsexpr)

(define-generics to-jsexpr
  (->jsexpr to-jsexpr)
  (->jsexpr/options wat to-jsexpr)
  (->jsexpr/bad)
  #:fast-defaults
  ())
