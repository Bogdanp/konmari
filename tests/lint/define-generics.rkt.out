/define-generics.rkt:12:4:warning:generic identifier 'to-jsexpr' not bound in definition
/define-generics.rkt:11:4:warning:identifier '->jsexpr/options' is never used
/define-generics.rkt:12:4:warning:identifier '->jsexpr/bad' is never used
/define-generics.rkt:9:18:warning:identifier 'to-jsexpr?' is never used
